doc/doxygen.conf.in

```diff
@@ -0,0 +1 @@
+EXCLUDE_PATTERNS += */src/*/*.cc
```

Comment:
`sconsUtils` will append this line to builds of the Stack-wide documentation. I don't think the Stack as a whole is ready for excluding source files; it would certainly need to be RFC'd.

I think `FILE_PATTERNS` doesn't get scoured from packages, so you might try

    FILE_PATTERNS = *.h *.py *.dox